python/pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[build-system]
+requires = ["setuptools", "setuptools-scm"]
+build-backend = "setuptools.build_meta"
+
+[project]
+name = "SeticorePy"
+version = "0.0.2"
+authors = [
+  { name="Kevin Lacker", email="lacker@gmail.com" },
+]
+description = "Python library related to seticore."
+readme = "README.md"
+requires-python = ">=3.7"
+classifiers = [
+    "Programming Language :: Python :: 3",
+    "License :: OSI Approved :: MIT License",
+    "Operating System :: OS Independent",
+]
+dependencies = [
+  "pycapnp",
+  "matplotlib",
+  "h5py",
+  "numpy",
+  "pandas",
+  "importlib_resources",
+  "PyQt5",
```

Comment:
why is PyQt5 a dependency?

Reply:
Ah! I was struggling with matplotlib complaining about Agg backends and not being able to plot... this was at `import SeticorePy.viewer` as the crosscheck.py was also inherently imported too... The TkAgg backend solution required more packages (AFAICT in my brief consideration) than the singular dependency of PyQt5 for the `Qt5Agg` backend or whatever. Yet I see that I kept related code external to the project... I'll remove the requirement.